config/configure_options/mpi_default

```diff
@@ -0,0 +1,103 @@
+#=====================================================================
```

Comment:
This is fine, apart from the name which suggests that it's the default that everybody should use to compile with mpi support. Not true; the oversubscribe flag is something specific that's only there for the github self tests so shouldn't this file be called mpi_for_github_selftests, say?

Reply:
I initially thought that the `--oversubscribe` flag wouldn't change the performance of programs that are not oversubscribed, but [that's not true](https://www.open-mpi.org/faq/?category=running#oversubscribing), so you make a good point. I've changed the name to `mpi_default_ci`.